dojo/unittests/tools/test_acunetix_parser.py

```diff
@@ -27,10 +27,11 @@ def test_parse_file_with_one_finding(self):
             # check endpoints
             self.assertEqual(1, len(finding.unsaved_endpoints))
             endpoint = finding.unsaved_endpoints[0]
+            endpoint.clean()
```

Comment:
```suggestion
            self.assertFalse(endpoint.is_broken())
```

Reply:
Could you make this modification in all unit tests?

Reply:
I'm not sure about removing `.clean()` because tests checking format of endpoints which will be stored (and they will be stored after calling [`.clean()`](https://github.com/DefectDojo/django-DefectDojo/pull/4473/files#diff-fe3c6e77f360d3f1156c8e78b208ac1608ccf54484123b42316731eb9122cecdR117))

But definitely I can add checks for `.is_broken()`

Reply:
We have a problem - `.is_broken` also checks if `product` is defined (because of [this](https://owasp.slack.com/archives/C014H3ZV9U6/p1621955497009900)) and endpoints coming from parser don't have defined products.

However, [`.is_broken == .clean() + check if product is defined`](https://github.com/DefectDojo/django-DefectDojo/pull/4473/files#diff-a797c37dd5bcb8a76fcf2ce34eb254ab52bbcbdea329b6f94080f96eeb6727feR1240), so calling `.clean()` without fail should be enough

I will revert last commits